functions/system.bash

```diff
@@ -471,6 +471,26 @@ memory_split() {
   else
     if echo "gpu_mem=16" >> /boot/config.txt; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
+
+  sed -i '/^dtoverlay=vc4-fkms-v3d/d' /boot/config.txt
```

Comment:
What exactly is this line doing, and why is it now needed?

Reply:
It's some graphics option for Pi4 only that appeared as part of Raspi OS, but it gets deleted now ("/d") because it's *not* needed .